### DISCUSSION TITLE
Hi, CEBRA team! How can I compare the neural dynamic or latent/decoding differences between two group of mice?

### DISCUSSION THREAD

**Discussion by qweasdf1354** [AdaptiveMotorControlLab/CEBRA#109] [Q&A] [answered]
Hi, CEBRA team! Thanks for **great algorisms** and **analysis methods**. 

I currently reading the **paper** and the **doc** in the web. I wanna to use the **great method** to my project of **two_photon + mice behaviour data**. At this time point, I got  the **[Neuron * Time] (row * column)** dataset and **continuous behaviour data (from DLC** and some user_defined characteristics like the **state of mice using discrete variables like 0 or 1**) from two group of mice (namely **Control** and **Experiment**). To be clarity, the behaviour paradigm is the same,and the only differences is the different group of mice.

**I wanna to know how can I compare the neural dynamic or latent/decoding differences between two group of mice?**

I have some idea for analysis the data. Could you help me to **exam** is the following idea **work** or **correct**? **Thanks a lot!**

There are **two hypothesis** I wanna to test from my data:

**One** is the control group of mice may decode the state of mice (user_defined characteristics using discrete variables like 0 or 1) **correctly** and the Experiment may decode **less correctly**. 
I wanna to do the following step to test above hypothesis. **1. Train models per mice. 2. Decoding for every mice. 3. Compare the KNN scores differences between Control and Experiment mice using t-test or non-parametric test.**

**The other** is **both of group of mcie** decode the state of mice **correctly** but the dynamic of neural activity is different.
I wanna to do the following step to test the this hypothesis. 1. Train models per mice as the same above. **2. compare the consistency** using the method your provide to test the consistency **between all mice from all group** that may find the **high consistency between the same group of mice** and **low consistency between the different of group of mice**. **3. Compare the training loss** between the following two model: one is **train from Control group**, one is t**rain from Experiment group**, one is **train from both Control and Experiment group**.

Thanks for your help! I am looking forward for your reply and appreciate for your help in advance.

**Comment by MMathisLab** [comment, marked as answer]
Thanks! 

Aside from maybe other statistical approaches, all the above sounds fine to me. 

You could additionally consider training with our multi-session (all control, all expt), then check decoding, consistency and goodness of fit (loss value).

**Comment by qweasdf1354** [comment]
Thanks!

**Comment by qweasdf1354** [comment]
Hi, CEBRA team! Thanks for your reply!
I have a question with multi-session training. 
I read the doc and it says 

> For now, multi-session training can only handle a unique set of continuous labels. All other combinations will raise an error.

Is that possible, I use the continuous_label to represent the discrete label? I mean using the 0 and 1 in continuous_label as the input for the cebra model.

Because the 0 and 1 may be the most important feature of this experiment.

Thanks a lot for help!
